### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by singingwolfboy**
This pull request has several commits that should increase the test coverage of this project. You may want to review this pull request one commit at a time.

### CODE REVIEW

File: tests/test_drafts.py [hunk 1]
```diff
@@ -23,10 +25,45 @@ def test_save_send_draft(api_client):
...
+    "mock_draft_saved_response", "mock_draft_deleted_response"
+)
+def test_delete_draft(api_client):
+    draft = api_client.drafts.create()
+    # Unsaved draft shouldn't throw an error on .delete(), but won't actually
+    # delete anything.
+    draft.delete()
+    # Now save the draft, and update the version so it's truthy
+    draft.save()
+    draft.version = 1
```
Comment (singingwolfboy) [inline tests/test_drafts.py]: Should this be necessary?

Comment (mhahnenberg) [inline tests/test_drafts.py]: Huh, very weird. Seems like a bug in the API at least. Maybe file an issue so we can revisit it later?

File: tests/test_accounts.py [hunk 2]
```diff
@@ -11,3 +12,28 @@ def test_create_apiaccount(api_client, monkeypatch):
...
+    account = api_client.accounts.create()
+    result = account.as_json()
+    assert isinstance(result, dict)
+
+
+@pytest.mark.xfail
+def test_account_upgrade(api_client, monkeypatch):
+    monkeypatch.setattr(api_client, "is_opensource_api", lambda: False)
+    account = api_client.accounts.create()
+    assert account.billing_state is False  # what should this be?
```
Comment (mhahnenberg) [inline tests/test_accounts.py]: cc @pfista

Comment (pfista) [inline tests/test_accounts.py]: This can be the following values:
```
'trial', 'trial-expired', 'developer-program', 'paid', 'cancelled', 'deleted'
```

**Pull request merged**